cpp/benchmarks/io/text/multibyte_split_benchmark.cpp

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
```

Comment:
```suggestion
 * Copyright (c) 2021, NVIDIA CORPORATION.
```
Many of the new files here are missing copyright headers.